Update hide-application-from-user-portal.md (#1123)
MicrosoftDocs/entra-docs: docs/identity/enterprise-apps/hide-application-from-user-portal.md

:::zone-end

> [!NOTE]

> > These instructions apply only to non first-party Microsoft Enterprise Applications. To learn more about first party Microsoft applications see this [link](https://learn.microsoft.com/troubleshoot/azure/entra/entra-id/governance/verify-first-party-apps-sign-in). Administrators also need to keep in mind that hiding the application from the end user does not prevent users signing in these applications via other then My Apps portal means (shared links, services dependencies, etc.).  

Reviewer: ```suggestion
> These instructions apply only to non first-party Microsoft Enterprise Applications. To learn more about first-party Microsoft applications see [First-party Microsoft applications in sign-in reports](/troubleshoot/azure/entra/entra-id/governance/verify-first-party-apps-sign-in). Administrators also need to keep in mind that hiding the application from the end user does not prevent users signing in these applications via other then My Apps portal means (shared links, services dependencies, etc). 
```

> > These instructions apply only to non first-party Microsoft Enterprise Applications. To learn more about first-party Microsoft applications see [First-party Microsoft applications in sign-in reports](/troubleshoot/azure/entra/entra-id/governance/verify-first-party-apps-sign-in). Administrators also need to keep in mind, that hiding the application from the end user does not prevent users signing in these applications via other than My Apps portal methods, such as shared links or service dependencies. 

Reviewer 2: ```suggestion
> These instructions apply only to non-first-party Microsoft Enterprise Applications. To learn more about first-party Microsoft applications see [First-party Microsoft applications in sign-in reports](/troubleshoot/azure/entra/entra-id/governance/verify-first-party-apps-sign-in). Administrators also need to keep in mind that hiding the application from the users doesn't prevent them from signing into these applications via methods other than the My Apps portal, such as shared links or service dependencies. 
```
